Primo Normalized XML.js

```diff
@@ -48,79 +47,79 @@ function detectImport() {
 function doImport() {
 	var doc = Zotero.getXML();
 	var ns = {
-		'p': 'http://www.exlibrisgroup.com/xsd/primo/primo_nm_bib',
-		'sear': 'http://www.exlibrisgroup.com/xsd/jaguar/search'
+		p: 'http://www.exlibrisgroup.com/xsd/primo/primo_nm_bib',
+		sear: 'http://www.exlibrisgroup.com/xsd/jaguar/search'
 	};
 	
 	var item = new Zotero.Item();
-	var itemType = ZU.xpathText(doc, '//p:display/p:type', ns)  || ZU.xpathText(doc, '//p:facets/p:rsrctype', ns) || ZU.xpathText(doc, '//p:search/p:rsrctype', ns);
+	var itemType = ZU.xpathText(doc, '//p:display/p:type', ns) || ZU.xpathText(doc, '//p:facets/p:rsrctype', ns) || ZU.xpathText(doc, '//p:search/p:rsrctype', ns);
 	if (!itemType) {
 		throw new Error('Could not locate item type');
 	}
 	
 	switch (itemType.toLowerCase()) {
-		case 'book':
-		case 'ebook':
-		case 'pbook' :
-		case 'books':
-		case 'score':
-		case 'journal':		//as long as we don't have a periodical item type;
- 			item.itemType = "book";
-			break;
-		case 'audio':
-		case 'sound_recording':
-			item.itemType = "audioRecording";
-			break;
-		case 'video':
-		case 'dvd':
-			item.itemType = "videoRecording";
-			break;
-		case 'computer_file':
-			item.itemType = "computerProgram";
-			break;
-		case 'report':
-			item.itemType = "report";
-			break;
-		case 'webpage':
-			item.itemType = "webpage";
-			break;
-		case 'article':
-		case 'review':
-			item.itemType = "journalArticle";
-			break;
-		case 'thesis':
-		case 'dissertation':
-			item.itemType = "thesis";
-			break;
-		case 'archive_manuscript':
-		case 'object':
-			item.itemType = "manuscript";
-			break;
-		case 'map':
-			item.itemType = "map";
-			break;
-		case 'reference_entry':
-			item.itemType = "encyclopediaArticle";
-			break;
-		case 'image':
-			item.itemType = "artwork";
-			break;
-		case 'newspaper_article':
-			item.itemType = "newspaperArticle";
-			break;
-		case 'conference_proceeding':
-			item.itemType = "conferencePaper";
-			break;
-		default:
-			item.itemType = "document";
-			var risType = ZU.xpathText(doc, '//p:addata/p:ristype', ns);
-			if (risType) {
-				switch (risType.toUpperCase()) {
-					case 'THES':
-						item.itemType = "thesis";
-						break;
-				}
+	case 'book':
```

Comment:
This indenting looks strange for me, but it was suggested by the linter...

Reply:
I've just changed this in the latest version of the config. The previous style is common in many languages, but there's now a [compelling reason](https://github.com/zotero/eslint-config-zotero/commit/aa9f19cf9bf479663e75b687d9ca320071d09d01) to indent switch cases in JavaScript, and the prevailing wisdom seems to have shifted.